API-Samples/Tutorial/markdown/10-init_render_pass.md

```diff
@@ -105,10 +111,28 @@ optimal layout so that the GPU can read it more efficiently.
 Conversions from one layout to another are called layout transitions
 in Vulkan.
 You have control over these transitions and can invoke them
-with an image memory barrier command or as a subpass dependency in a render
-pass.
-In this case, the sample code uses image memory barrier commands, which
-are detailed later in this section.
+in one of three ways:
+
+1. Memory Barrier Command (via `vkCmdPipelineBarrier`)
+1. Render Pass specification
+1. Render Pass subpass specification
+
+The memory barrier command is an explicit layout transition command
+that you place in a command buffer.
+For example, you might place one in a command buffer to convert a buffer from
+optimal to linear after executing a `vkCmdCopyBuffer` command
```

Comment:
Using optimal and linear makes it sound like you are talking about tiling, and buffers are their own class of tiling / format feature, and I know of no way to change the tiling of an image after it's created.  I would use the words "layout from DEST_OPTIMAL to ???_OPTIMAL" instead.